Dockerfile

```diff
@@ -14,7 +14,13 @@ RUN dnf update -y && \
     jq \
     openssl-devel \
     pip \
-    python3-devel && \
+    python3-devel \
+    dnf-plugins-core && \
+    dnf clean all
+
+RUN dnf copr enable -y chmouel/tektoncd-cli && \
+    dnf install -y \
+    tektoncd-cli && \
```

Comment:
Can we use the `tkn` `ClusterTask` instead to ensure compatibility with the cluster running the pipelines?

Reply:
No, because in `finally` block tasks always run in parallel so we can't use tekton task to fetch and another task for uploading. It needs to be in one task which means that we need all our dependencies in one image + script in one task.

Reply:
I changed the uploading task and broke it into 2 steps and used an official tekton image. So I dropped a tekton cli installation and made corresponding changes is https://github.com/redhat-openshift-ecosystem/operator-pipelines/pull/26